asreview/webapp/api/projects.py

```diff
@@ -114,17 +121,38 @@ def _get_authenticated_folder_id(project_id, user):
     if (project_from_db and user == project_from_db.owner) or (
         project_from_db is None
     ):
-        user_uuid = uuid5(NAMESPACE_URL, str(user.id)).hex
+        uuid = _get_project_uuid(project_id, user.id)
```

Comment:
Could you change `uuid` to something like `project_uuid`? That way it doesn't conflict with the builtin `uuid` module.

Reply:
Done